.binder/requirements.txt

```diff
@@ -1 +1 @@
-etna[all] @ git+https://github.com/tinkoff-ai/etna.git@master
+etna[all] @ git+https://github.com/etna-team/etna.git@master
```

Comment:
It isn't very useful to see this changes here. Could you update the branch `internal_datasets` for it to disappear?